ee/modules/110-istio/hooks/generate_password_test.go

```diff
@@ -6,17 +6,51 @@ Licensed under the Deckhouse Platform Enterprise Edition (EE) license. See https
 package hooks
 
 import (
-	"encoding/json"
+	"encoding/base64"
+	"fmt"
 
 	. "github.com/onsi/ginkgo"
 	. "github.com/onsi/gomega"
 
+	"github.com/deckhouse/deckhouse/go_lib/hooks/generate_password"
 	. "github.com/deckhouse/deckhouse/testing/hooks"
 )
 
 var _ = Describe("Modules :: istio :: hooks :: generate_password ", func() {
-	f := HookExecutionConfigInit(`{"istio":{"internal":{}, "auth": {}}}`, `{"istio":{}}`)
-	Context("without external auth", func() {
+	var (
+		hook = generate_password.NewBasicAuthPlainHook(moduleValuesKey, authSecretNS, authSecretName)
+
+		testPassword    = "t3stPassw0rd"
+		testPasswordB64 = base64.StdEncoding.EncodeToString([]byte(
+			fmt.Sprintf("admin:{PLAIN}%s", testPassword),
+		))
+
+		// Namespace should be created before creating the Secret.
+		nsManifest = `
```

Comment:
Why do we need to create a namespace for tests?

Reply:
IDK, but listing secrets returns empty list without creating namespace first.

Reply:
I don't think this is true. It does not match with how fake client works.

Reply:
Something was wrong on first runs, now it is ok without creating namespace. I've removed all this workarounds.